client/homebrew/editor/editor.jsx

```diff
@@ -292,8 +356,11 @@ const Editor = createClass({
 	},
 
 	brewJump : function(targetPage=this.getCurrentPage()){
```

Comment:
Instead of setting and unsetting a global `scrollingJump` everywhere, I think it might make more sense to just add a parameter to `brewJump()` and `sourceJump()` that determines if the scroll is smooth or instant, with a default of smooth. So something like

```suggestion
	brewJump : function(targetPage=this.getCurrentPage(), smooth=true){
```